providers/axfrddns/axfrddnsProvider.go

```diff
@@ -133,7 +132,7 @@ func initAxfrDdns(config map[string]string, providermeta json.RawMessage) (provi
 	} else if len(api.nameservers) != 0 {
 		api.master = api.nameservers[0].Name + ":53"
 	} else {
-		return nil, fmt.Errorf("nameservers list is empty: creds.json needs a default `nameservers` or an explicit `master`")
+		return nil, errors.New("nameservers list is empty: creds.json needs a default `nameservers` or an explicit `master`")
```

Comment:
Why is this line changed to `errors.New` while lines 206, 223, 227, 287, and 387 still use `fmt.Errorf`?

Reply:
Good question!

errors.New() is appropriate when there is no formatting needed.

* fmt.Errorf(): Has "printf" semantics
* errors.New(): has "print" semantics